packages/libp2p/src/connection-manager/index.ts

```diff
@@ -571,7 +583,7 @@ export class DefaultConnectionManager implements ConnectionManager, Startable {
   async acceptIncomingConnection (maConn: MultiaddrConnection): Promise<boolean> {
```

Comment:
Please can you add some tests that show that incoming connections are denied for multiaddrs in the deny list and allowed for ones in the allow list.

They should show that individual addresses and ranges are handled correctly in both IPv4 and IPv6 formats.

It should be a case of adding more tests like [this one](https://github.com/libp2p/js-libp2p/blob/main/packages/libp2p/test/connection-manager/index.spec.ts#L144-L171) and [this one](https://github.com/libp2p/js-libp2p/blob/main/packages/libp2p/test/connection-manager/index.spec.ts#L75-L91).